.github/workflows/release-embedding-sdk.yml

```diff
@@ -86,9 +84,23 @@ jobs:
         with:
           m2-cache-key: "release-sdk"
 
+      - name: Update readme
+        run: |
+          ./bin/embedding-sdk/release_utils.bash update_readme ${{ inputs.sdk_version }}
+
       - name: Bump published npm package version
         run: |
-          sed -i -E 's/("version": ").*"/\1${{ inputs.sdk_version }}"/' enterprise/frontend/src/embedding-sdk/package.template.json
+          ./bin/embedding-sdk/release_utils.bash update_package_json_template ${{ inputs.sdk_version }}
+
+      - name: Upload release utils script, so it can be used later on when checking out `master` branch
```

Comment:
we could remove this step after the script is merged to master, but for now, let's copy this file over, because we'll need it again when checking out `master` later on.